org.opent2t.sample.binaryswitch.superpopular/com.insteon.binaryswitch/js/thingTranslator.js

```diff
@@ -69,9 +69,20 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
         if: ['oic.if.a', 'oic.if.baseline']
     };
     
+    // Build the connectionStatus resource (read-only)
+    var connectionStatus = {
```

Comment:
I like this solution, and it will work well for device graph updates.  Since this is additive, I see no issue with merging it, but we need to update the RAML (at least) and all of the other translators that will support this resource before it can be consumed.